packages/contracts/contracts/optimistic-ethereum/bridge/L1CrossDomainMessenger.sol

```diff
@@ -37,6 +59,18 @@ contract L1CrossDomainMessenger is IL1CrossDomainMessenger, BaseCrossDomainMesse
      * Public Functions
      */
 
+    /**
+     * Sets the target messenger address.
+     * @param _targetMessengerAddress New messenger address.
+     */
+    function setTargetMessengerAddress(
+        address _targetMessengerAddress
+    )
+        public
+    {
+        targetMessengerAddress = _targetMessengerAddress;
+    }
+
```

Comment:
This looks duped between the L1 and L2 Messengers. Any reason why it was removed from the Base?

Reply:
Ah yeah, that's because both of the messengers use the `ICrossDomainMessenger` interface. I could change the `BaseCrossDomainMessenger` to inherit from the `ICrossDomainMessenger` instead, actually...